src/buildApplication/generateFunctionsMap.ts

```diff
@@ -43,6 +44,8 @@ export async function generateFunctionsMap(
 		functionsDir
 	);
 
+	await checkInvalidFunctions(processingResults);
```

Comment:
@james-elicx 
![Screenshot 2023-04-06 at 22 12 53](https://user-images.githubusercontent.com/61631103/230495440-3e1fa817-46f1-4677-94d6-c012ac060468.png)

Reply:
Yeah, sorry about that, I was using the commit changes thing on the GitHub website for some of your suggestions and I had to change that one manually, just pushed a commit which includes that fix as well addressing some other suggestions

Reply:
ah no problem it happens :sweat_smile: , sorry about making suggestions that can break your code :sweat_smile: :bow: